samples/csharp/getting-started/MulticlassClassification_HeartDisease/HeartDiseaseConsoleApp/HeartTextLoaderFactory.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace MulticlassClassification_HeartDisease
+{
+    public static class HeartTextLoaderFactory
+    {
+        public static TextLoader CreateTextLoader(MLContext mlContext)
+        {
+            TextLoader textLoader = mlContext.Data.CreateTextReader(
```

Comment:
Instead of explicetely providing all the columns to the TextLoader, since they are quite a few (14 columns), could you load it through the data-class, as shown here?:

`var dataView = mlContext.Data.ReadFromTextFile<InspectedRow>(dataPath, hasHeader: true);`

So you use the observation data-class (`InspectedRow`, in this case) for providing all the columns info.


https://blogs.msdn.microsoft.com/dotnet/2019/01/10/announcing-ml-net-0-9-machine-learning-for-net/#additional-api-improvements-in-mlnet-09

Or here:
https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md#how-do-i-verify-the-model-quality